drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -31,7 +31,8 @@ def projects(**args)
     organizations_field :organizations
 
     def organizations(**args)
-      resolve_organizations(Hmis::Hud::Organization.all, **args)
+      # todo @martha - don't understand why this is failing the test but not manifesting in qa-hmis-2
```

Comment:
@gigxz sorry for the delay. I got swept down an unrelated rabbit hole.

- after we paired, your recommendation was to "add tests for checking against data source and organization" - which I still have a todo for [here](https://github.com/greenriver/hmis-warehouse/pull/4788/files#diff-d700a2b25aff90f1650d8cabf6ae2a496773aa4b668e12dc579c0e340a430413R283) and will add comments about below
- while I was working on trying to find where I would put those tests / if they already exist, I noticed that here in the organizations query, we weren't limiting scope to viewable_by the current user. This is bug, right? I reproduced it in an rspec test below, fixed the bug, and the test passes.
- BUT, it's a mystery to me why this bug isn't manifesting on qa-hmis-2. I visted the [organizations page](https://qa-hmis-2.openpath.host/projects) there and I don't see all the organizations from qa-hmis. Maybe this is not really a bug and I'm missing something?

Reply:
Yeah good question, this code jumps through some hoops. It's because the viewable_by scope is already applied in the `HasOrganizations` concern. I think we use that pattern in several places, so I think we can leave it as is. The `user` args defaults to `current_user`:

https://github.com/greenriver/hmis-warehouse/blob/f550cdd12e2ad040c59c774056850bc55ce4d0de/drivers/hmis/app/graphql/types/hmis_schema/has_organizations.rb#L36-L41